includes/sanitizers/class-amp-tag-and-attribute-sanitizer.php

```diff
@@ -559,6 +559,15 @@ private function process_node( DOMElement $node ) {
 			}
 		}
 
+		// Remove the element if it has a invalid layout attributes.
+		if (
+			! $this->is_valid_layout( $tag_spec, $node ) &&
+			$this->remove_invalid_child( $node, [ 'code' => 'invalid_layout' ] )
+		) {
+			$this->remove_node( $node );
```

Comment:
Note that `remove_invalid_child` will have already removed the node, so this `$this->remove_node( $node );` should not be here.